baseservice/iodevices/common.py

```diff
@@ -0,0 +1,28 @@
+from typing import BinaryIO, Dict, Any, Optional
+
+MessageHeaders = Dict[str, Any]  # this is the type for message metadata
+
+
+class Message:
```

Comment:
`dataclass`/`pydantic`?

Reply:
hate dataclasses... it requires you to know more then the code it saves.

Reply:
We'll wait till you will add another class that contains only data and I'll reopen this conversation :)